src/util/ecdsa.rs

```diff
@@ -37,6 +37,23 @@ pub struct PublicKey {
 }
 
 impl PublicKey {
+    /// Constructs compressed ECDSA PublicKey from the provided generic Secp256k1 public key
```

Comment:
Very minor nit: if it's not code, i.e. a type, I wouldn't write public key in camel case. If it's code/a type it should be formatted as such.